packages/client/src/store/index.js

```diff
@@ -24,4 +28,15 @@ export default new Vuex.Store({
     organization,
     tenants,
   },
+  state: {
+    alerts: {},
+  },
+  mutations: {
+    addAlert(state, alert) {
+      Vue.set(state.alerts, randomId(), alert);
+    },
+    dismissAlert(state, alertId) {
+      Vue.delete(state.alerts, alertId);
+    },
+  },
```

Comment:
nit: please implement as a separate module (like the other parts of GOST store are) rather than adding at top-level.

Reply:
makes sense!